hack/fake-apiserver/go.mod

```diff
@@ -110,3 +110,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/metal3-io/cluster-api-provider-metal3/api => ./../../api
```

Comment:
This does not work, the ../../.. is outside container build context for our image builds.

We have open issue about it here: https://github.com/metal3-io/cluster-api-provider-metal3/issues/2157

Reply:
can you elaborate? I didn't get the container build context issue from the link? I can build the image locally just fine

Reply:
Added details in mentioned issue: https://github.com/metal3-io/cluster-api-provider-metal3/issues/2157#issuecomment-2703769154

Reply:
Thanks @tuminoid for the context. I was building the CAPM3 image and than works fine but fcas image build fails as you pointed out.